websites/T/TuneIn/presence.ts

```diff
@@ -1,85 +1,119 @@
 const presence = new Presence({
-		clientId: "844108776793178122",
-	}),
-	strings = presence.getStrings({
-		play: "general.playing",
-		pause: "general.paused",
-		live: "general.live",
-	});
-
-let title, author;
+	clientId: "844108776793178122",
+});
 
 presence.on("UpdateData", async () => {
-	const presenceData: PresenceData = {
-		largeImageKey:
-			"https://cdn.rcd.gg/PreMiD/websites/T/TuneIn/assets/logo.png",
-		startTimestamp: Math.floor(Date.now() / 1000),
-	};
+	const settings = {
+			lang: await presence.getSetting("lang"),
+			timestamps: await presence.getSetting("timestamps"),
+			cover: await presence.getSetting("cover"),
+			private: await presence.getSetting("privacy"),
+		},
+		strings = await presence.getStrings({
+			play: "general.playing",
+			pause: "general.paused",
+			browse: "general.browsing",
+			live: "general.live",
+			listening: "general.listeningMusic",
+		}),
```

Comment:
This may create unnecessary requests for fetching strings each cycle. It also doesn't take into account your `lang` setting.
Check how other presences do this for an example, such as Amazon Music